baxter_sim_kinematics/src/arm_kinematics.cpp

```diff
@@ -34,6 +34,9 @@
 #include <cstring>
 #include <ros/ros.h>
 #include <baxter_sim_kinematics/arm_kinematics.h>
+#if ROS_VERSION_MINIMUN(1, 14, 0)  //Melodic
```

Comment:
I'm so sorry, but my original comment was typo.
`ROS_VERSION_MINIMUN` -> `ROS_VERSION_MINIMUM`
can you correct this?

Reply:
Thank you!